isaric/parsers/brazil-followup.toml

```diff
@@ -0,0 +1,829 @@
+  name = "headache"
+  date = { field = "flw_survey_date" }
+  start_date = { field = "flw_survey_date", apply = { function = "startDate", params = [
+    7,
+  ] } }
+  phase = "followup"
+  duration_type = "event"
+  is_present = { field = "flw1_headache", ref = "Y/N" }
+
+[[observation]]
+  name = "history_of_fever"
+  start_date = { field = "flw_first_sym" }
+  date = { field = "flw_first_sym", apply = { function = "endDate", params = [
+    14,
+  ] } }
+  duration_type = "event"
+  phase = "initial_illness"
+  is_present = { field = "flw_first_14d___1", ref = "checkbox" }
+
+[[observation]]
+  name = "history_of_fever"
+  start_date = { field = "flw_survey_date", apply = { function = "startDate", params = [
+    7,
+  ] } }
+  date = { field = "flw_survey_date" }
+  duration_type = "event"
+  phase = "followup"
+  if = { flw_last_fever = 1 }
+  is_present = { field = "flw_fever", ref = "Y/N" }
+
+[[observation]]
+  name = "history_of_fever"
+  start_date = { field = "flw_survey_date", apply = { function = "startDate", params = [
+    14,
+  ] } }
+  date = { field = "flw_survey_date", apply = { function = "startDate", params = [
+    7,
+  ] } }
+  duration_type = "event"
+  phase = "followup"
+  if = { flw_last_fever = 2 }
+  is_present = { field = "flw_fever", ref = "Y/N" }
+
+[[observation]]
+  name = "history_of_fever"
+  start_date = { field = "flw_survey_date", apply = { function = "startDate", params = [
+    28,
+  ] } }
+  date = { field = "flw_survey_date", apply = { function = "startDate", params = [
+    14,
+  ] } }
+  duration_type = "event"
+  phase = "followup"
+  if = { flw_last_fever = 3 }
+  is_present = { field = "flw_fever", ref = "Y/N" }
+
+[[observation]]
+  name = "history_of_fever"
+  start_date = { field = "flw_survey_date", apply = { function = "startDate", params = [
+    60,
+  ] } }
+  date = { field = "flw_survey_date", apply = { function = "startDate", params = [
+    30,
+  ] } }
+  duration_type = "event"
+  phase = "followup"
+  if = { flw_last_fever = 4 }
+  is_present = { field = "flw_fever", ref = "Y/N" }
+
+[[observation]]
+  name = "history_of_fever"
+  start_date = { field = "flw_survey_date", apply = { function = "startDate", params = [
+    90,
+  ] } }
+  date = { field = "flw_survey_date", apply = { function = "startDate", params = [
+    60,
+  ] } }
+  duration_type = "event"
+  phase = "followup"
+  if = { flw_last_fever = 5 }
+  is_present = { field = "flw_fever", ref = "Y/N" }
+
+[[observation]]
+  name = "inability_to_walk_scale"
+  date = { field = "flw_survey_date" }
+  start_date = { field = "flw_survey_date", apply = { function = "startDate", params = [
+    7,
+  ] } }
+  phase = "followup"
+  duration_type = "event"
+  value = { field = "flw_walking_today", ref = "inabilityWalkToday" }
+
+[[observation]]
+  name = "inability_to_walk_scale"
+  date = { field = "flw_survey_date" }
+  start_date = { field = "flw_survey_date", apply = { function = "startDate", params = [
+    7,
```

Comment:
params 7 can be removed as this data refers to day of survey completion (also pending outcome of discussion with ISARIC re. EQ5D license and use of this data as commented in the Tier 1 review)